src/bin/ion/commands/inspect.rs

```diff
@@ -222,17 +239,13 @@ impl<'a, 'b> IonInspector<'a, 'b> {
 
             match item {
                 SystemStreamItem::SymbolTable(lazy_struct) => {
-                    let is_append = lazy_struct.get("imports")?
-                        == Some(ValueRef::Symbol(SymbolRef::with_text("$ion_symbol_table")));
-                    if !is_append {
-                        next_symbol_id = 10; // First available SID after system symbols in Ion 1.0
-                    }
-                    self.inspect_symbol_table(next_symbol_id, lazy_struct)?;
```

Comment:
🗺️ Now that a `LazyValue` can return a reference to its `SymbolTable`, the symbol ID assignment logic can be localized to the method where it's displayed.